### DISCUSSION TITLE
Duda T1 sobre casos borde compras y ventas

### DISCUSSION THREAD

**Discussion by tsunayoshi21** [IIC2133-PUC/2024-1#71] [Tarea 1] [answered]
Holaa, saben si en la 1 de la tarea hay que considerar el caso en que un usuario ponga más de 1 compra (o venta)? En ese caso, habría que actualizar el monto?, o añadirlo como nodo nuevo?

**Comment by sachondo7** [comment, marked as answer]
Cada compra/venta es una línea nueva de input, por lo que se considera un nodo nuevo
